changes/watch_darwin.go

```diff
@@ -0,0 +1,84 @@
+// heavily inspired by https://github.com/fsnotify/go.fsevents
```

Comment:
I should probably remove that repo from the fsnotify organization so it doesn't trip anyone up.